webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,15 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand")
+    .on("input", function() {
+      $(this).css({"overflow":"hidden", "height": "scrollHeight+px"});
```

Comment:
First, I tried running the code with `"height": this.scrollHeight + "px"`. It did not change the height. Then I tried `"scrollHeight+px"`. The height increased fine but did not adjust when we delete a few lines. Could you please explain me what am I doing wrong here?